hr_attendance_management/migrations/10.0.2.0.0/pre-migration.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2019 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Nicolas Badoux <n.badoux@hotmail.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+from openupgradelib import openupgrade
+from datetime import date
+
+
+@openupgrade.migrate(use_env=True)
+def migrate(env, version):
+    if not version:
+        return
+    cr = env.cr
+
+    # compute extra hours lost until last cron execution
+    last_computation = env['ir.config_parameter'].get_param(
+            'hr_attendance_management.last_balance_cron_execution',
```

Comment:
I think this won't work because last_balance_cron_execution don't exist at pre-migration or did I miss something? Are we suppose to add this in our database before migrating?

Reply:
correct, we will always fall back to the first january of this year